src/runtime/forester/flow.rs

```diff
@@ -291,6 +329,34 @@ pub fn monitor(
     }
 }
 
+// Handle ticking a flow node with the state "Halting".
+// Returns a tuple of the new state and the cursor position of the child to be halted, if one exists.
+pub fn halt(flow_type: &FlowType, tick_args: RtArgs) -> (RNodeState, Option<usize>) {
+    match flow_type {
+        FlowType::Sequence
+        | FlowType::MSequence
+        | FlowType::RSequence
+        | FlowType::Fallback
+        | FlowType::RFallback => {
+            // Sequence/fallback nodes check if they need to halt a child at the RUNNING_CHILD cursor position.
+            let running_child_cursor = tick_args
+                .find(RUNNING_CHILD.to_string())
+                .and_then(RtValue::as_int)
+                .and_then(|v| Some(v as usize));
```

Comment:
`.and_then(|v| Some(v as usize))` better to replace to `.map(|v| v as usize)`

Reply:
That makes sense, fixed.